config/jobs/etcd/etcd-presubmits.yaml

```
@@ -67,6 +67,7 @@ presubmits:
         args:
         - -c
         - |
+          set -euo pipefail
           make verify
           make fix
```

If adding the pipefail makes the job report the error correctly, we'll need the following lines to work correctly. This addresses the issue with not being able to run `golangci-lint`.

```suggestion
          export PATH=$GOPATH/bin:$PATH && make verify
          export PATH=$GOPATH/bin:$PATH && make fix
```

---

+1

Thanks @ivanvc

---

Added -thanks!